### PR TITLE
Add test for pki-server upgrade

### PR DESCRIPTION
**Pull request by edewata**
A new test has been added to verify the `pki-server upgrade` tool using a basic upgrade script.

https://github.com/edewata/pki/blob/upgrade/docs/development/Creating-Upgrade-Script.adoc

### CODE REVIEW

File: docs/development/Creating-Upgrade-Script.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+class <name>(pki.server.upgrade.PKIServerUpgradeScriptlet):
+
+    def __init__(self):
+        super(<name>, self).__init__()
+        self.message = ...
+
+    ...
+----
+
+Make sure the class name matches the `<name>` in the filename.
```
Comment (ckelleyRH) [inline docs/development/Creating-Upgrade-Script.adoc:43]: Can you use `super(__name__, self).__init__()` to automatically get the right name?

**Review by ckelleyRH — APPROVED** [review]
LGTM, thanks for making this! One doc suggestion.

**Review by fmarco76 — APPROVED** [review]
LGTM

File: docs/development/Creating-Upgrade-Script.adoc (hunk 1 above)
Comment (edewata) [reply docs/development/Creating-Upgrade-Script.adoc:43]: I actually had to use `super().__init__()` instead. I've updated the doc and the example. Thanks!

**Comment by edewata** [comment]
@ckelleyRH @fmarco76 Thanks!

**Pull request merged**